apps/apps.json

```diff
@@ -118,5 +118,11 @@
     "description": "Zsa is a keyboard firmware and configuration tool.",
     "script": "zsa.sh",
     "asRoot": true
+  },
+  {
+    "name": "jq",
```

Comment:
Not sure if this should be an independent app. Indeed, it can be a dependency of other installers. If you check the `init.sh` file you can see that it is already part of the installation as prerequisites.
https://github.com/blacksrc/dot/blob/main/init.sh#L18
We might want to close this PR. WDYT?